cmdstanpy/stanfit.py

```diff
@@ -1323,6 +1371,28 @@ def variational_params_dict(self) -> OrderedDict:
         """Returns inferred parameter means as Dict."""
         return OrderedDict(zip(self.column_names, self._variational_mean))
 
+    def stan_variable(self, name: str) -> np.ndarray:
+        if name not in self.stan_vars_dims:
+            raise ValueError('unknown name: {}'.format(name))
+        col_idxs = list(self._metadata.stan_vars_cols[name])
+        vals = list(self._variational_mean)
+        xs = [vals[x] for x in col_idxs]
+        shape = ()
+        if len(col_idxs) > 0:
+            shape = self._metadata.stan_vars_dims[name]
+        return np.ndarray(
+            shape=shape, buffer=np.array(xs)
+        )
```

Comment:
I've never seen this form of creating an ndarray before - does it have an advantage over `np.array(xs).reshape(shape)`? Merely curious - the docs seem to suggest `np.ndarray` is primarily an internal constructor, and my assumption is it does two allocations?

Reply:
this is solely due to my lack of fluency in Python.  thanks for catching this.

Reply:
This still needs to be resolved here and in the other new functions